### ISSUE TITLE
Rule depending on bookmarks

### ISSUE THREAD

**Issue by xrad** [kintesh/containerise#117]
I'm currently moving from Chrome to FF. To (kind-of) match Chrome/Google profiles, I use FF containers. I have also imported bookmarks from different Chrome/Google profiles into different bookmark hierarchies. Can (could?) Containerise open a URL in a specific Container based on the location in the bookmarks tree? What about duplicate entries? Or am I using all this the wrong way?

**Comment by LoveIsGrief** [comment]
Hi, that isn't how Containerise works. I'm not against it, but I'm not entirely sure how it would work. 

 - Would the URL directly match a bookmark or would it match the domain?
 - Duplicate bookmarks would open in whichever container matched first

Could you explain why you'd like it to work this way?

**Comment by xrad** [comment]
Yeah I'm not sure either. :) Maybe I'm just too much focused on the bookmarks side of things.

Previously I've been using Chrome with different Google profiles. Each profile ran in a different window. Each had it's own set of bookmarks. So my thinking was to simply to try tying bookmarks to FF containers. 

I've played around a bit more now and I now use the Sticky Window Containers Add-On which more or less emulates my previous setup - I'm used to having containers clearly seperated via different windows so that works quite well for me. The main visible difference to my previous setup is now that I see all bookmarks in all windows. Previously I only saw the per-profile bookmarks. But I guess I can live with this. 

So with this setup I suppose I don't really need Containerise and the feature I asked for.
